src/ersilia_pack/linter.py

```diff
@@ -1,9 +1,16 @@
 import os
 import json
 import argparse
+import logging
+
+
 
 from . import BasePacker
 
+logging.basicConfig(level=logging.INFO, 
+                    format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
```

Comment:
Let's put the logger creation logic in the `utils.py` file, and import the `logger` from there. This is so that we don't have to put the same code in all the files.

Reply:
Ok, will do that. Thank you .